includes/validate.php

```diff
@@ -112,11 +112,10 @@ function edac_validate( $post_ID, $post, $action ) {
 	do_action( 'edac_after_get_content', $post_ID, $content, $action );
 
 	if ( ! $content['html'] ) {
-		add_option( 'edac_password_protected', true );
+		update_option( 'edac_password_protected', true );
 		return;
-	} else {
-		delete_option( 'edac_password_protected' );
 	}
+	delete_option( 'edac_password_protected' );
```

Comment:
* Replaced `add_option` with `update_option`
* The `else` is not necessary since the previous condition bails early